changes/428.feature

```diff
@@ -0,0 +1 @@
+- Modify the loading process so that the scheduler can be loaded reflecting scheduler_opts.
```

Comment:
You don't have to put the starting `- ` since `* ` will be automatically added by [towncrier](https://pypi.org/project/towncrier/) when generating the changelog. Please remove it. :wink: